src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -105,1002 +98,1361 @@
+            if (i == 0 || !pColumnVisible[i]) {
+                jTableInventoryRow.getColumnModel().getColumn(i).setMinWidth(0);
+                jTableInventoryRow.getColumnModel().getColumn(i).setMaxWidth(0);
+                jTableInventoryRow.getColumnModel().getColumn(i).setPreferredWidth(0);
+            }
+        }
+        DefaultTableCellRenderer centerRenderer = new DefaultTableCellRenderer();
+        centerRenderer.setHorizontalAlignment(SwingConstants.CENTER);
+        jTableInventoryRow.getColumnModel().getColumn(3).setCellRenderer(centerRenderer);
+    }
+
+    private List<MedicalInventoryRow> getMedicalInventoryRows(String code) throws OHServiceException {
+        List<MedicalInventoryRow> inventoryRowsList = new ArrayList<>();
+        List<MedicalWard> medicalWardList = new ArrayList<>();
+        Medical medical;
+        MedicalInventoryRow inventoryRowTemp;
+        if (code != null) {
+            medical = medicalBrowsingManager.getMedicalByMedicalCode(code);
+            if (medical != null) {
+                medicalWardList = movWardBrowserManager.getMedicalsWard(wardId, medical.getCode(), false);
+            } else {
+                MessageDialog.error(null, MessageBundle.getMessage("angal.inventory.noproductfound.msg"));
+            }
+        } else {
+            medicalWardList = movWardBrowserManager.getMedicalsWard(wardId, false);
+        }
+        for (MedicalWard medicalWard: medicalWardList) {
+            inventoryRowTemp = new MedicalInventoryRow(0, medicalWard.getQty(), medicalWard.getQty(), null,
+                medicalWard.getMedical(), medicalWard.getLot());
+            if (!existInInventorySearchList(inventoryRowTemp)) {
+                inventoryRowsList.add(inventoryRowTemp);
+            }
+        }
+        return inventoryRowsList;
+    }
+
+    private boolean existInInventorySearchList(MedicalInventoryRow inventoryRow) {
+        boolean found = false;
+        List<MedicalInventoryRow> invRows = inventoryRowSearchList.stream()
+                .filter(inventoryRow1 -> inventoryRow1.getMedical().getCode().equals(inventoryRow.getMedical().getCode())).toList();
+        if (!invRows.isEmpty()) {
+            for (MedicalInventoryRow invR: invRows) {
+                if (inventoryRow.getLot() != null && invR.getLot() != null) {
+                    if (inventoryRow.getLot().getCode().equals(invR.getLot().getCode())) {
+                        found = true;
+                        break;
+                    }
+                } else {
+                    if (invR.getLot() == null && inventoryRow.getLot() == null) {
+                        found = true;
+                        break;
+                    }
+                }
+            }
+        }
+        return found;
+    }
+
+    private JRadioButton getSpecificRadio() {
+        if (specificRadio == null) {
+            specificRadio = new JRadioButton(MessageBundle.getMessage("angal.inventory.specificproduct.radio"));
```

Comment:
I don't know where the `radio` suffix came from but all over `JRadioButton` message labels end in `btn`.   It is only in this PR that `radio` is used to the best of my searching.